src/config/toml_reader.c

```diff
@@ -134,13 +135,13 @@ bool readFTLtoml(struct config *oldconf, struct config *newconf,
 }
 
 // Parse TOML config file
-static toml_table_t *parseTOML(void)
+static toml_table_t *parseTOML(const unsigned int version)
 {
 	// Try to open default config file. Use fallback if not found
 	FILE *fp;
-	if((fp = openFTLtoml(GLOBALTOMLPATH, "r")) == NULL)
+	if((fp = openFTLtoml("r", version)) == NULL)
 	{
-		log_warn("No config file available (%s), using defaults",
+		log_info("No config file available (%s), using defaults",
```

Comment:
This logging is unconditionally, but should be conditionally. Otherwise it logs

```
2023-11-21 20:40:12.676 [15792M] INFO: No config file available (No such file or directory), using defaults
2023-11-21 20:40:12.676 [15792M] INFO: No config file available (No such file or directory), using defaults
2023-11-21 20:40:12.684 [15792M] INFO: Config file written to /etc/pihole/pihole.toml
```

Instead, it should log something like

```
INFO: Config file /etc/pihole/pihole.toml not available, trying to restore backup
INFO: Config file /etc/pihole/config_backup/pihole.toml.1 not available, trying to restore next backup
...
INFO: No config file and no config backup file not available, using defaults
INFO: Config file written to /etc/pihole/pihole.toml

``